New version: Jello v0.1.4

JuliaRegistrator opened the pull request:
- Registering package: Jello
- Repository: https://github.com/paulxshen/Jello.jl
- Created by: @paulxshen
- Version: v0.1.4
- Commit: d0df8530c09f7976ed5fe22c768aaa773637582d
- Git reference: HEAD

DilumAluthge commented:
[noblock]

Looks like there is a merge conflict.

@paulxshen Can you re-trigger Registrator? This will fix the merge conflict.

JuliaHubRegistrator committed:
New version: Jello v0.1.4

UUID: 6872b481-e419-48a0-81d2-be4ee5684529
Repo: https://github.com/paulxshen/Jello.jl.git
Tree: 7f2af10ab00bc37325649559254ca19dc65aa3c5

Registrator tree SHA: 7242ef64be5953fac9bf8b3efa3a7c4d2d44ca09
